### PR TITLE
Remove deprecated EventForwarder from common-utils

### PR DESCRIPTION
**Pull request by ChumpChief**
EventForwarder was moved to client-utils from common-utils 6 months ago and deprecated in its original location at the time.  This change would remove the deprecated version in common-utils.

AB#7092

### CODE REVIEW

File: .changeset/fresh-chairs-wait.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+---
+"@fluidframework/common-utils": major
+---
+
+EventForwarder removed from common-utils
+
+The EventForwarder class in common-utils was deprecated and has now been removed.
```
Comment (Josmithr) [inline .changeset/fresh-chairs-wait.md:7]: Is there any migration guidance we should offer here for affected users?

Comment (Josmithr) [reply .changeset/fresh-chairs-wait.md:7]: If not, we should probably note that explicitly.

**Review by Josmithr — APPROVED** [review]

**Comment by Josmithr** [comment]
@tylerbutler any idea why the github bot removed the breaking change tag?

File: .changeset/fresh-chairs-wait.md (hunk 1 above)
Comment (ChumpChief) [reply .changeset/fresh-chairs-wait.md:7]: Yeah it's a bad pattern I'm trying to remove so no replacement recommended.  I'll update the changeset.

**Comment by ChumpChief** [comment]
> @tylerbutler any idea why the github bot removed the breaking change tag?

Maybe because it's an `@internal` API?  Though with the new definition of `@internal` it should prob still be considered a breaking change.

**Comment by Josmithr** [comment]
> > @tylerbutler any idea why the github bot removed the breaking change tag?
> 
> Maybe because it's an `@internal` API? Though with the new definition of `@internal` it should prob still be considered a breaking change.

New definition of internal? I wasn't aware of any changes there,

**Comment by tylerbutler** [comment]
> @tylerbutler any idea why the github bot removed the breaking change tag?

It's a bug in the workflow config. I tried to fix it in https://github.com/microsoft/FluidFramework/pull/19020 but clearly wasn't successful.

**Pull request merged**